### PR TITLE
Sending cheeps

### PR DESCRIPTION
**Pull request by yacobbakiz**
It is now possible to add cheeps that are written to both the usertimeline and the public timeline.

### CODE REVIEW

File: src/Chirp.Razor/Pages/Public.cshtml.cs [hunk 1]
```diff
@@ -7,6 +7,9 @@ public class PublicModel : PageModel
 {
     private readonly ICheepService _service;
     public List<CheepDto> Cheeps { get; set; } = new List<CheepDto>();
+    [BindProperty]
+    public string Message { get; set; }
```
Comment (Yous113) [inline src/Chirp.Razor/Pages/Public.cshtml.cs:15]: Non-nullable property 'Message' must contain a non-null value when exiting constructor. Consider adding the 'required' modifier or declaring the property as nullable.

File: src/Chirp.Razor/Pages/SubmitMessage.cs [hunk 2]
```diff
@@ -0,0 +1,32 @@
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+
+namespace Chirp.Razor.Pages;
+
+public class SubmitMessageModel : PageModel
+{
+    private readonly ICheepService _service;
+    [BindProperty]
+    public string Message { get; set; }
```
Comment (Yous113) [inline src/Chirp.Razor/Pages/SubmitMessage.cs:13]: Same should be non-null

File: src/Chirp.Razor/Pages/UserTimeline.cshtml.cs [hunk 3]
```diff
@@ -8,6 +8,8 @@ public class UserTimelineModel : PageModel
     private readonly ICheepService _service;
     public List<CheepDto> Cheeps { get; set; } = new List<CheepDto>();
     public Author? Author { get; set; }
+    [BindProperty]
+    public string Message { get; set; }
```
Comment (Yous113) [inline src/Chirp.Razor/Pages/UserTimeline.cshtml.cs:15]: same here

**Review by alwaysDebuggging — COMMENTED** [review]
My dear colleague Yous made some good comments.

**Pull request merged**